common/src/types.rs

```diff
@@ -139,3 +139,10 @@ pub enum Commit {
     ExtraAgendaTransaction(ExtraAgendaTransaction),
     ChatLog(ChatLog),
 }
+
+/// The special finalization proof commit in the `pf` branch.
```

Comment:
at the below, it says `fp` branch. Is `fp` branch and `pf` branch is individual?